Rankings.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>Rankings</title>
+		<link rel="stylesheet" href="C:\Users\Asus\Desktop\rankingsCSS.css">
+	</head>
+	<body>
+		<header><h1>Rankings</h1></header>
+		<table>
+			<tr>
+				<th>User name</th>
+				<th>Score of user</th>
+				<th>Rank of user</th>
+			</tr>
+			% for name, rank, score in people :
+			<tr>
+				<td>{{name}}</td>
+				<td>{{score}}</td>
+				<td>{{rank}}</td>
+			</tr>
+			<tr>
```

Comment:
you don't have to repeat the block again and again. That's what the loop is for.